### PR TITLE
React Components

### PR DESCRIPTION
**Pull request by Kandy-Peter**
For this pull request I added new functionalities in the app:

1. I created a new folder called components
2. Create a new file `Calculator.js` in that directory
3. Create the calculator template and display it at `http//:localhost:300` to see the result.
4. Design the template using SCSS.
5. Install React/Redux linter and fix errors.

### CODE REVIEW

**Review by dasileker — COMMENTED** [review]
### **Hi @Kandy-Peter ,** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no workflow and no linters check running. so please fix those and resubmit again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,70 +1,43 @@
...
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+### `npm start`
 
-### Making a Progressive Web App
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+The page will reload when you make changes.\
```
Comment (Lameck1) [inline README.md]: - [x] Please write a proper getting started section. It should be about how to get started with your own project and not the `Create React App`. This section should detail how one can get a local copy of your project, install dependencies and properly run the project locally on a computer.

- [x] May you also ensure that your README has all the sections suggested in this [Microverse README template](https://github.com/microverseinc/readme-template) for students.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @Kandy-Peter, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,70 +1,55 @@
...
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Frist Microverse projects to get start with React.
+The math magician web app is a simple calculator for solving minor math problems.
 
-## Available Scripts
+## Screenshoot
 
-In the project directory, you can run:
+![Screeshot](./src/Screenshot.png)
```
Comment (moise10r) [inline README.md]: - [OPTIONAL ] The screenshot picture does not display correctly, Kindly fix this.

**Review by moise10r — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @Kandy-Peter** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**